server/routes/todos.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const router = express.Router();
+const {
+  addTodos,
+  getTodos,
+  getTodoDetail,
+  editTodos,
+  deleteTodos,
+  changeCompleted,
+} = require("../controller/todoController");
+
+router.use(express.json());
```

Comment:
요건 사소하다고 볼 수도 있는데요~
require문은 require문끼리 묶어두고, 그 이후에 코드를 작성하는게 해당 파일이 사용하고 있는 외부 의존성을 파악하기 편하기 때문에 아래와 같이 작성하는 것을 저는 추천드립니다~
```suggestion
const express = require("express");

const {
  addTodos,
  getTodos,
  getTodoDetail,
  editTodos,
  deleteTodos,
  changeCompleted,
} = require("../controller/todoController");

const router = express.Router();
router.use(express.json());
```